src/towncrier/_builder.py

```diff
@@ -435,6 +436,9 @@ def get_indent(text: str) -> str:
         top_underline=top_underline,
         get_indent=get_indent,  # simplify indentation in the jinja template.
         issues_by_category=issues_by_category,
+        **(
```

Comment:
Just asking ?

Would it be possible to always have `header_prefix="#" * md_header_level`

If there is no extra level, we will have just empty string.

I hope that this will make it easier to write custom templates as you don't have to check for missing  `header_prefix` in Jinja template data

Reply:
True, jinja doesn’t care if the template doesn’t use some variables in the context.